lib/asciidoctor/converter/factory.rb

```diff
@@ -53,7 +53,9 @@ def default initialize_singleton = true
             new ::Concurrent::Hash.new
```

Comment:
But isn't Asciidoctor.js providing Concurrent::Hash so that this LoadError isn't hit in the first place?